src/components/Thanks/ShareChallenge.vue

```diff
@@ -0,0 +1,247 @@
+						</button>
+						<button
+							class="tw-flex tw-items-center tw-justify-center tw-gap-1 tw-border tw-border-gray-800 tw-rounded tw-py-1 tw-basis-1/3"
+							v-kv-track-event="[
+								'post-checkout',
+								'share',
+								'linkedin',
+								utmCampaign,
+								loanId
+							]"
+							@click="showSharePopUp(
+								linkedInShareUrl({ utmCampaign, utmContent }),
+								'Thanks for sharing to LinkedIn!'
+							)"
+						>
+							<kv-icon name="linkedin" title="LinkedIn" style="width: 26px; fill: #0A66C2;" />
+							<span class="tw-font-medium">LinkedIn</span>
+						</button>
+						<button
+							class="tw-flex tw-items-center tw-justify-center tw-gap-1 tw-border tw-border-gray-800 tw-rounded tw-py-1 tw-basis-1/3"
+							:class="copyStatus.class"
+							:disabled="copyStatus.disabled"
+							v-kv-track-event="[
+								'post-checkout',
+								'share',
+								'copy-link',
+								utmCampaign,
+								loanId
+							]"
+							@click="copyUrl({ utmCampaign, utmContent }, copyStatus.text)"
+						>
+							<kv-material-icon
+								:icon="mdiLink"
+								style="width: 28px;"
+								class="tw-bg-secondary tw-p-0.5 tw-rounded"
+							/>
+							<span class="tw-font-medium">{{ copyStatus.text }}</span>
+						</button>
+						<!-- eslint-enable max-len -->
+					</div>
+				</div>
+			</div>
+		</kv-grid>
+	</kv-page-container>
+</template>
+<script>
+import KvProgressCircle from '@/components/Kv/KvProgressCircle';
+import RewardsIcon from '@/assets/icons/inline/rewards.svg';
+import socialSharingMixin from '@/plugins/social-sharing-mixin';
+import { mdiLink } from '@mdi/js';
+import { getFullUrl } from '@/util/urlUtils';
+import clipboardCopy from 'clipboard-copy';
+import KvIcon from '@/components/Kv/KvIcon';
+import KvPageContainer from '~/@kiva/kv-components/vue/KvPageContainer';
+import KvGrid from '~/@kiva/kv-components/vue/KvGrid';
+import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
+
+export default {
+	name: 'ShareChallenge',
+	inject: ['apollo'],
+	mixins: [socialSharingMixin],
+	components: {
+		KvProgressCircle,
+		KvMaterialIcon,
+		RewardsIcon,
+		KvIcon,
+		KvPageContainer,
+		KvGrid
+	},
+	props: {
+		goal: {
+			type: Object,
+			default: () => ({}),
+		},
+		teamPublicId: {
+			type: String,
+			default: '',
+		},
+		loan: {
+			type: Number,
+			default: 0,
+		},
+		lender: {
+			type: Object,
+			default: () => ({}),
+		},
+		isGuest: {
+			type: Boolean,
+			default: false
+		},
+	},
+	data() {
+		return {
+			utmCampaign: 'teamschallenge',
+			mdiLink,
+			copyStatus: {
+				class: '',
+				disabled: false,
+				text: 'Copy link'
+			}
+		};
+	},
+	computed: {
+		loanId() {
+			return this.loan?.id ?? '';
+		},
+		borrowerName() {
+			return this.loan?.name ?? '';
+		},
+		teamName() {
+			return this.goal?.name ?? '';
+		},
+		teamChallengePath() {
+			return this.teamPublicId ? `/team/challenge/${this.teamPublicId}` : '/teams';
```

Comment:
Should this return the user to `/lend/filter` with team parameter?

Reply:
Yeah we should make this change, since this component is only for the new challenge, and we won't show it until the new challenge is live.